docs/source/tutorial/interfaces.rst

```diff
@@ -667,9 +665,9 @@ Given the following list:
     testList : List Nat
     testList = [3,4,1]
 
-We can sort it using the default ``Ord`` implementation, by using the ``sort``
-function available with ``import Data.List``, then we can try with the named
-implementation ``myord`` as follows, at the Idris prompt:
+We can sort it using the default ``Ord`` implementation by using the ``sort``
+function, available via ``import Data.List``. Then, we can try with the named
+implementation ``myord`` — both are shown below at the Idris prompt:
```

Comment:
I agree with most of these changes, but the original comma after "implementation" should definitely be preserved; it helps the flow of the sentence. Another em-dash.

Reply:
The prepositional phrase "by using the sort function…” is restrictive, therefore the comma is technically incorrect. A variant could be "We can sort it using the default ``Ord`` implementation. This can be done by using the ``sort`` function, available via ``import Data.List``.

The em-dash was the only way I found to reconcile the fact that both prompts are shown after this paragraph, even though the first one relates to the first sentence. The original was a single sentence, but had its own issues.

Reply:
It just reads weird to me without it – too much uninterrupted flow/stream-of-words. :shrug:

I think the dash is fine, it should just be an en-dash rather than an em-dash (or the spaces should be removed on either side of the em-dash, but then that _is_ very American)